sdk/samples/iot/generate_certificate.ps1

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+openssl ecparam -out device_ec_key.pem -name prime256v1 -genkey
+openssl req -new -days 365 -nodes -x509 -key device_ec_key.pem -out device_ec_cert.pem -config x509_config.cfg -subj "/CN=paho-sample-device1"
+openssl x509 -noout -text -in device_ec_cert.pem
+
+if (Test-Path device_cert_store.pem)
+{
+  Remove-Item device_cert_store.pem
+}
+
+Get-Content device_ec_cert.pem, device_ec_key.pem | Set-Content device_cert_store.pem
+
+Write-Output "IMPORTANT:"
```

Comment:
Nit: If someone is using Linux I don't think they need to be told this.  Could you wrap with an if ($IsWindows) and if ($IsMac) (if there is such a thing)?

Reply:
Also if you can put into OS specific, I'd recommend going with a full on `Write-Warning` that'll make this pop even more instead `Write-Output`